packages/api/src/schema/db-schema.yaml

```diff
@@ -715,6 +715,9 @@ components:
         deleted:
           type: boolean
           description: Set to true when stream deleted
+        routeToCDN:
+          type: boolean
+          description: If stream should be routed through CDN
```

Comment:
Currently it's 0.0-100.0 float number in catalyst-api https://github.com/livepeer/catalyst-api/blob/main/main.go#L101 .
We might want it in here instead of boolean

Reply:
Yeah makes sense 👍